documentation/modules/ROOT/pages/connectors/sqlserver.adoc

```diff
@@ -167,23 +167,22 @@ For example, consider a SQL Server installation with an `inventory` database tha
 * `fulfillment.dbo.customers`
 * `fulfillment.dbo.orders`
 
-ifdef::community[]
 === Schema change topic
 
+For a table for which CDC is enabled, the {prodname} SQL Server connector stores the history of schema changes to that table in a database history topic. This topic reflects an internal connector state and you should not use it. If your application needs to track schema changes, there is a public schema change topic. The name of the schema change topic is the same as the logical server name specified in the connector configuration.
+
 [WARNING]
 ====
-The format of the schema change topic messages is in incubating state and it can change without further notice.
+The format of messages that a connector emits to its schema change topic is in an incubating state and can change without notice.
 ====
 
-SQL Server connector stores the historic schema structure of database tables in a database history topic.
-This topic should be considered as an internal state of the connector and should not be used by the user.
-If the application needs to track changes in the source database there is the public-facing schema change topic.
-The topic name is the same as the logical server name configured in connector configuration.
+{prodname} emits a message to the schema change topic when:
 
-{prodname} emits a new message to this topic whenever a new table is streamed from or when the structure of the table is altered ({link-prefix}:{link-sqlserver-connector}#sqlserver-schema-evolution[schema evolution procedure must be followed]).
-The message contains a logical representation of the table schema.
+* You enable CDC for a table. 
```

Comment:
Is this part correct?  If yes, then we need to change the descriptions of `type` in the logical representation. That is, does CREATE indicate that CDC has been enabled for a table?  The way it is documented now, there is a message emitted to the schema change topic when you create a table. The doc does not indicate any relationship to tables for which the connector is capturing changes. Likewise, does DROP mean that CDC was disabled for a table or that the table was deleted?

Reply:
If I understand what you're asking, whether the table is CDC enabled or not is implicit.  

During streaming, we fetch a list of all tables that are CDC enabled and from there we filter that list based on include and exclude configuration filters.  This list is then used when schema changes are detected to avoid emitting events to the schema history topic for tables that shouldn't be there, e.g. a table that isn't monitored.

So for `CREATE`, the table would have already had to have had CDC enabled.  For a `DROP`, that raises a question which I don't have an answer for without testing is whether that explicitly drops CDC functionality.  For an `ALTER`, the table is already CDC enabled and is being modified.